src/App.js

```diff
@@ -1,13 +1,22 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
 import Header from "./Components/Header";
 import "./styles/main.scss";
 import Landing from "./Pages/Landing";
-
+import { TweenMax } from "gsap";
+import { Switch, Route } from "react-router-dom";
+import Register from "./Pages/Auth/Register";
 function App() {
+  let app = useRef(null);
+  useEffect(() => {
+    TweenMax.to(app, 0, { css: { visibility: "visible" } });
+  });
   return (
-    <div className="App">
+    <div className="App" ref={(el) => (app = el)}>
       <Header />
-      <Landing />
+      <Switch>
+        <Route exact path="/" component={Landing} />
```

Comment:
Can your remove these routing too? Its not related to the card

Reply:
> You haven't changed the Header.js and Landing.js file extension to jsx

I changed it, and  then it threw an error that bit could not find the file

Reply:
You cancel and restart your server